web/js/codeworld_shared.js

```diff
@@ -70,6 +70,72 @@ var Alert = (() => {
   return mine;
 })();
 
+var hintBlacklist = [
+    // Symbols that only exist to implement RebindableSyntax or map to
+    // built-in Haskell types.
+    "Bool",
+    "IO",
+    "fail",
+    "fromCWText",
+    "fromDouble",
+    "fromInt",
+    "fromInteger",
+    "fromRational",
+    "fromString",
+    "ifThenElse",
+    "toCWText",
+    "toDouble",
+    "toInt",
+
+    // Deprecated exports.
+    "MousePress",
+    "MouseRelease",
+    "MouseMovement",
+    "MouseButton",
+    "LeftButton",
+    "RightButton",
+    "MiddleButton",
+    "Maybe",
+    "Nothing",
+    "Just",
+    "withDefault",
+    "hasValue",
+    "definitely",
+    "path",
+    "thickPath",
+    "text",
+    "styledText",
+
+    // Experimental exports.
+    "White",
+    "Black",
+    "Gray",
+    "Grey",
+    "Red",
+    "Orange",
+    "Yellow",
+    "Green",
+    "Blue",
+    "Purple",
+    "Pink",
+    "Brown",
+    "Pi",
+];
+
+var codeWorldBuiltinsDocs = {};
+
+function onHover(cm, data, node){
+    var result = null;
```

Comment:
[nit] I think it would be cleaner to just add two returns, and not use the local var.

Reply:
Ok.